DependencyInjection/SensioFrameworkExtraExtension.php

```diff
@@ -32,6 +32,10 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.xml');
 
+        if (class_exists('Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface')) {
+            $loader->load('psr7.xml');
+        }
```

Comment:
All listeners are currently disablable (does this word actually exist?) in this bundle. So it should be the case here too. the default value could depend on the availability of the bridge.